cloud/disk_manager/internal/pkg/services/disks/create_empty_disk_task_test.go

```diff
@@ -258,3 +258,69 @@ func TestCancelCreateEmptyDiskTaskBeforeRunIsCalled(t *testing.T) {
 	mock.AssertExpectationsForObjects(t, storage, nbsFactory, execCtx)
 	require.NoError(t, err)
 }
+
+func TestCreateEmptyLocalDiskTask(t *testing.T) {
+	ctx := context.Background()
+	storage := storage_mocks.NewStorageMock()
+	nbsFactory := nbs_mocks.NewFactoryMock()
+	nbsClient := nbs_mocks.NewClientMock()
+	execCtx := newExecutionContextMock()
+	cellSelector := cells_mocks.NewCellSelectorMock()
+
+	params := &protos.CreateDiskParams{
+		BlocksCount: 123,
+		Disk: &types.Disk{
+			ZoneId: "zone",
+			DiskId: "disk",
+		},
+		BlockSize: 456,
+		Kind:      types.DiskKind_DISK_KIND_SSD_LOCAL,
+		CloudId:   "cloud",
+		FolderId:  "folder",
+		AgentIds:  []string{"agent"},
+	}
+	task := &createEmptyDiskTask{
+		storage:      storage,
+		nbsFactory:   nbsFactory,
+		params:       params,
+		state:        &protos.CreateEmptyDiskTaskState{},
+		cellSelector: cellSelector,
+	}
+
+	// TODO: Improve this expectations.
+	storage.On("CreateDisk", ctx, mock.Anything).Return(&resources.DiskMeta{
+		ID: "disk",
+	}, nil)
```

Comment:
```suggestion
	storage.On("CreateDisk", ctx, mock.Anything).Return(
		&resources.DiskMeta{ID: "disk"},
		nil,
	)
```

Кажется, что так будет читаться лучше